### PR TITLE
Added category service to client

### PR DESCRIPTION
**Pull request by Maxim-Zavidei**
# Added category service to client

### CODE REVIEW

File: src/Client/Services/CategoryService/CategoryService.cs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+using BlazorEcommerce.Shared;
+
+namespace Client.Services.CategoryService;
+
+public class CategoryService : ICategoryService
+{
+
+    private readonly HttpClient _http;
+    public List<Category> Categories { get; set; } = new List<Category>();
+
```
Comment (LilianBittar) [inline src/Client/Services/CategoryService/CategoryService.cs:11]: why space?

**Review by LilianBittar — COMMENTED** [review]
it's ok

File: src/Client/Services/CategoryService/CategoryService.cs (hunk 1 above)
Comment (Maxim-Zavidei) [reply src/Client/Services/CategoryService/CategoryService.cs:11]: https://thumbs.gfycat.com/AnnualEquatorialGardensnake-max-1mb.gif

Comment (Maxim-Zavidei) [reply src/Client/Services/CategoryService/CategoryService.cs:11]: becauese i wanto so 🤌🏽

**Review by LilianBittar — APPROVED** [review]

**Pull request merged**